crypto/dilithium/ml_dsa.c

```diff
@@ -32,6 +32,14 @@ int ml_dsa_65_keypair(uint8_t *public_key   /* OUT */,
   return (crypto_sign_keypair(&params, public_key, private_key) == 0);
 }
 
+int ml_dsa_65_keypair_internal(uint8_t *public_key   /* OUT */,
```

Comment:
I'm not sure I understand why we need to define and expose the internal functions?

Reply:
We need to expose them for ACVP testing, as part of the process of FIPS validation.